mlir/lib/Dialect/Vector/Transforms/VectorTransforms.cpp

```diff
@@ -1043,6 +1047,144 @@ class ExtractOpFromElementwise final
   }
 };
 
+static bool isSupportedMemSinkElementType(Type type) {
```

Comment:
[nit] It would be good to document this. Also, why not `isSupportedMemSinkElementType` -> `isMulitByteScalar`? Ultimately, that's what this is checking, right?